api_v3/services/UserService.php

```diff
@@ -150,6 +166,11 @@ public function updateAction($userId, KalturaUser $user)
 			}			
 			$dbUser = $user->toUpdatableObject($dbUser);
 			$dbUser->save();
+			if (($user->id || $user->screenName) &&
+				categoryKuserPeer::isCategroyKuserExistsForKuser($dbUser->getId()))
+			{
+				$this->updateCategoryKuser($dbUser->getId(),$user);
```

Comment:
That should be triggered from the kuser postUpdate and not from the service, a user id or screen name could be changed from PS2, internal scripts or maintenance scripts.